samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -29,9 +29,11 @@
        Ex: dotnet user-secrets set "EmbeddingConfig:Key" "MY_EMBEDDING_KEY" */
   },
 
-  // Directory from which to load semantic skills
+  // Directory from which to load semantic skills, if used
   "SemanticSkillsDirectory": "", // Ex: ".\\SemanticSkills" (relative to executable)
 
+  "KeyVaultName": "", // Name of the Key Vault for secrets, if used - Ex: "myvault"
```

Comment:
I recommend changing this to KeyVaultURI. When we get outside of the Azure Public cloud, the URIs change (e.g., government and sovereign clouds).

Reply:
Doing so!